lib/core/util.js

```diff
@@ -167,7 +167,7 @@ module.exports = {
   destroy,
   bodyLength,
   isBuffer,
-  queueMicrotask: global.queueMicrotask || (cb => Promise.resolve()
+  queueMicrotask: global.queueMicrotask /* istanbul ignore next */ || (cb => Promise.resolve()
```

Comment:
Could we just do `global.queueMicrotask = null` in a test?

Reply:
Mh...we could but I definitely don't like playing with global.
Also, since we only support Node and we are on >10, do we really need this?

Reply:
I would remove this completely. `queueMicrotask` should be available in 12+, and we should stop support v10 in the next major anyway.

Reply:
I'm good with that

Reply:
Removed in separate PR. Ignore it here.